### PR TITLE
Create atmel-samg55j19a-c.md

### PR DESCRIPTION
**Pull request by brianjang**
a document for Atmel SAMG55 + WINC1500 instructions.
- hardware : Atmel SAMG55, Atmel WINC1500
- using Atmel studion7 IDE to create Azure IoT project 
- using Azure IoT C-SDK

### CODE REVIEW

**Comment by azurecla** [comment]
Hi __@brianjang__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        In order for us to evaluate and accept your PR, we ask that you sign a contribution license agreement. It's all electronic and will take just minutes. I promise there's no faxing. https://cla.azure.com.
    </span>

TTYL, AZPRBOT;

File: doc/get_started/atmel-samg55j19a-c.md [hunk 1]
```diff
@@ -0,0 +1,102 @@
...
+<a name="Build"></a>
+# Step 3: Build and Run the sample
+
+<a name="Load"></a>
+## 3.1 Build SDK and sample
+
+<a name="buildrunapp"/>
+## Build and Run the sample project via Atmel Studio7: 
+1. double click "Ateml_Azure_EXAMPLE.atsln" : **```TBD```**
+2. Open the ```simplesample_http.c``` file in the Atmel Studio7.
```
Comment (tameraw) [inline doc/get_started/atmel-samg55j19a-c.md]: Where is the package that brought in this code?

**Comment by azurecla** [comment]
__@brianjang__, Thanks for signing the contribution license agreement so quickly! Actual humans will now validate the agreement and then evaluate the PR.
<br />Thanks, AZPRBOT;

**Comment by brianjang** [comment]
for "how to update firmware and Flash the root certificate of the Azure IoT hub host"
=> you can get "WINC1500 Firmware Update Project(v19.4.4)" from Atmel studio's example project.
=> if you want to get "`WINC1500 Firmware Update Project(v19.4.4)"'s code please refer to Step1 ~ Step 3

for "Ateml_Azure_EXAMPLE.atsln"
=> this is Atmel studio project for Azure IoT, but we did not released it public area.
=> may be we can send this project file to you, but we need to make a decision that how to deploy this project with my company.
=> if this part is critical point while you are review our PR then please let us know.

**Comment by ewertons** [comment]
Hi Brian, thanks a lot for this document, it looks good. 
Could you add a few changes before we merge it back into the Azure repo please?
Here are the changes:

1. The section "Instructions for using this template" seems to be directed to the developer writting the documentation, and not to the final customer. It should be removed;
2. These two comments are shown on the browser as one line each, with no line break. Was that the intent of "#####"? If not please check it;
	"Set up your IoT hub. #### Create a device on IoT Hub"
	"Usage document. #### Write down device credentials"
3. Could you fix the letter case on the lines throught the doc to start with upper case?
	Like "how to update firmware and Flash the root..." should start as "How to update...";
4. Steps 1 to 4 under section "how to update firmware and Flash the root certificate of the Azure IoT hub host" under Step 2 are a little confusing. Could you write just the numbers for the sub steps, or use letters for the sub steps?
5. Just for matching the format we have, could you change "windows" to "Windows"? Thanks;
6. Under "Build and Run the sample project via Atmel Studio7",
	a. What is the "TBD" on step 1 (double click "Ateml_Azure_EXAMPLE.atsln" :  TBD)?
	b. the numbering of the steps is not continuous, could you change them to be 1 to 7?

Also, where are "Ateml_Azure_EXAMPLE.atsln", the main.h with the WLAN connection macros?
I didn't run the sample on the board yet because I didn't find them.

**Comment by brianjang** [comment]
Hi Ewertons.

Thank you for your effort.

We modify "atmel-samg55j19a-c.md" document to reflect your guide (from 1 ~ 6)

I would like to add one comment for 
-  Also, where are "Ateml_Azure_EXAMPLE.atsln", the main.h with the WLAN connection macros?
I didn't run the sample on the board yet because I didn't find them. 

we are working for embedding "MS Azure IoT ASF project" into our Atmel Studio Framework.
- please let me explain our plan for this task.
- if you confirm that our implementation is correct with "WINC1500_AzureIoT_CONNECT_EXAMPLE1_160318.zip" project which is passed to Hector.

so could you please test it with "WINC1500_AzureIoT_CONNECT_EXAMPLE1_160318.zip"?

**Comment by ewertons** [comment]
A few more comments:

1. "If you need to setup your SAMG55 device, please refer the getting started instructions here ."
    It might be better as 'If you need to setup your SAMG55 device, please refer to the "Getting Started" instructions here.'
2. "Build and Run the sample project via Atmel Studio7:" -> the numbered list is broken (numbers are not in sequence)
3. "this document is located in src/doc from the Step 3" -> do you mean "this document is located in 'src/doc' on the project created on previous step"? as it's written it's a little confusing, there is no "Step 3".
4. I tried uploading the cert to the device, but the steps are broken, I could not find 'samd21_xplained_pro_certificate_update.bat' - could you check if it's missing please?

Thanks,
Ewerton

**Comment by brianjang** [comment]
for ""Build and Run the sample project via Atmel Studio7:" -> the numbered list is broken (numbers are not in sequence)"

=> in my view, there is no issue. the numbered list is correct (from 1 ~ 5)
could you tell me more detailed descriptins?

**Comment by ewertons** [comment]
I'm visualizing the md file on Edge, it shows like this:
capture

Interesting. Firefox shows the list with the right numbers. It seems to be a bug on Edge :(
------------------
I found one more item that needs to be fixed on the doc.

"Select the MS Azure ToT project corresponding to your Xplained Pro board and then press OK button to import MS Azure ToT project".
I believe it should be "MS Azure IoT" (not TOT).

It should be "Select the MS Azure IoT project corresponding to your Xplained Pro board and then press OK button to import MS Azure IoT project".

**Comment by tameraw** [comment]
@brianjang - Closing as there is no update on the code review comments. Feel free to reopen once feedback addressed, or submit a new PR if you think it would be better. Thanks.

**Comment by brianjang** [comment]
@tameraw - if we ready to add our project file to Atmel studio then i will request it to reopen.
Thank you.

**Comment by spincraft** [comment]
Thanks for your efforts on this project.
May I join this?
Thanks,
Don

**Comment by brianjang** [comment]
@spincraft 

Sure. Please enjoy it.

Thank you for your request.

**Pull request closed without merging**